blob/gcsblob/gcsblob.go

```diff
@@ -189,8 +196,16 @@ func openBucket(ctx context.Context, client *gcp.HTTPClient, bucketName string,
 	if bucketName == "" {
 		return nil, errors.New("gcsblob.OpenBucket: bucketName is required")
 	}
+	clientOpts := []option.ClientOption{option.WithHTTPClient(useragent.HTTPClient(&client.Client, "blob"))}
```

Comment:
Same here, use an `if/else`. Right now you have `option.WithHTTPClient` twice, the behavior is probably undefined.

Reply:
The later will override the former, but let just re-assign to `clientOpts`.